syncserver-db-common/src/error.rs

```diff
@@ -8,14 +8,14 @@ use thiserror::Error;
 /// Error specific to any MySQL database backend. These errors are not related to the syncstorage
```

Comment:
Hmm. Technically, we should only pull this file in if we have some SQL database we're using, so the `lib.rs` should only include it if appropriate feature flags are set. That could add a lot of flag logic though...

Reply:
I think that fb10b2c is good